### ISSUE TITLE
Simpletests for drupal core are failing in travis even if all tests pass.

### ISSUE THREAD

**Issue by yanniboi** [LionsAd/drupal_ti#60]
I am running all drupal core simpletests with my module enabled, and I was really confused to see travis failing my build even though all the tests were passing. It turns out that the regex is catching some of the drupal test descriptions:
```
egrep -i "([1-9]+ fail)|(Fatal error)|([1-9]+ exception)" /tmp/simpletest-result.txt && exit 1
```
```
UserLoginTest.php  Drupal\user\Tests\UserLoginTest
Raw "There have been more than 3 failed login attempts for this account."
```
"[1-9]+ fail" catches "3 failed"

**Comment by yanniboi** [comment]
a37901c91bd7811871cfc01ec6ede480b2cdeb4e

**Comment by LionsAd** [comment]
#61 fixed this and will be in 1.5.0.
